templates/default/initscript.erb

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Created by chef
+
+### BEGIN INIT INFO
+# Provides: Bitcoin Core
+# Required-Start: $networking
+# Required-Stop: $networking
+# Default-Start: 2 3 4 5
+# Default-Stop: 0 1 6
+# Short-Description: Start at boot and stop at shutdown
+# Description: Starts Bitcoin Core service
+### END INIT INFO
+
+DAEMON="<%= @bitcoind %>"
+ARGS=""
+PIDFILE="<%= @btc_home %>/bitcoind.pid"
+
+case "$1" in
+    start)
+        echo "Starting server"
+        cd /tmp
+        /sbin/start-stop-daemon --start --pidfile $PIDFILE \
```

Comment:
Debian-ism but I think it's fine since Redhat-like distros have another way to install it via the `package` recipe.